packages/cli/src/commands/create.ts

```diff
@@ -21,15 +22,10 @@ export class CreateCotar extends Command {
       logger.error({ output: outputFile }, 'Output file exists, aborting..');
       return;
     }
-
-    if (!args.inputFile.endsWith('.tar')) {
-      logger.error({ input: args.inputFile }, 'Input file must be a .tar file');
```

Comment:
Not only tar file now?

Reply:
we have a lot of `.covt` files that I cannot index without renaming them :(